### ISSUE TITLE
New Rule: reports the unsupported import.meta properties.

### ISSUE THREAD

**Issue by ota-meshi** [eslint-community/eslint-plugin-n#419]
### Rule details

The `n/no-unsupported-features/node-builtins` rule reports when `require.resolve()` is not available, but does not report `import.meta.resolve()`.
I would like a rule that reports when `import.meta.resolve()` is not available.

### What type of rule is this?

Warns about a potential problem

### Example code

```js
import.meta.resolve(); // Reports if it is not available for the specified Node version.
import.meta.filename; // Reports if it is not available for the specified Node version.
import.meta.dirname; // Reports if it is not available for the specified Node version.
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

**Comment by scagood** [comment]
I am in favour of this, it's something I have hit a few times too 😅
